index.html

```diff
@@ -40,13 +40,165 @@ <h1 class="headline-job">I'm a software developer</h1>
             <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hestiate to contact me.</p>
             <ul class="socials">
-                <li><a href="#"><img src="./Images//twitter.png" alt=""></a></li>
-                <li><a href="#"><img src="./Images//linkedin.png" alt=""></a></li>
-                <li><a href="#"><img src="./Images//medium.png" alt=""></a></li>
-                <li><a href="#"><img src="./Images//github.png" alt=""></a></li>
-                <li><a href="#"><img src="./Images//angellist.png" alt=""></a></li>
+                <li><a href="#"><img src="./Images//twitter.svg" alt=""></a></li>
+                <li><a href="#"><img src="./Images//linkedin.svg" alt=""></a></li>
+                <li><a href="#"><img src="./Images//medium.svg" alt=""></a></li>
+                <li><a href="#"><img src="./Images//github.svg" alt=""></a></li>
+                <li><a href="#"><img src="./Images//angellist.svg" alt=""></a></li>
             </ul>
         </div>
     </section>
+    <button class="btn-disabled"><img src="./Images//Disabled.svg" alt=""></button>
+    <section>
+        <h1 class="head-recent">My Recent <br> Works</h1>
+        <span class="recent-bar"></span>
+        <div class="grid-container">
+            <div class="grid-item">
+              <div class="flex-container">
+                    <div class="works-image"></div>
+                    <div class="works-inner">
+                        <p>Multi-Post Stories<br>Gain+Glory</p>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">css</a></li>
+                            <li><a href="#">javScript</a></li>
+                            <li><a href="#">html</a></li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
```

Comment:
- [ ] As requested by the previous reviewer kindly make changes to the spacing and border. Use this border `1px solid #D0D9D4;` to match the Fimga design
- [ ] Kindly implement in all the cards as requested by the previous reviewer.
### Your design
![your](https://user-images.githubusercontent.com/81727988/212473448-9666793a-0f1b-4b7d-81e4-07ee2bba319b.jpg)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/212473452-025e7b12-8c18-4ff7-ac6c-2302c6b12e08.jpg)